apps/src/templates/levelSummary/summary.module.scss

```diff
@@ -75,7 +75,7 @@ label.sectionSelector {
   float: right;
 }
 
-[dir="rtl"] .toggleContainer {
+[dir='rtl'] .toggleContainer {
```

Comment:
These quote changes happened on autosave... assuming this is OK (although distracting for reviewers)